modin/pandas/test/test_groupby.py

```diff
@@ -58,7 +59,49 @@
 # have too many such instances.
 # TODO(https://github.com/modin-project/modin/issues/3655): catch all instances
 # of defaulting to pandas.
-pytestmark = pytest.mark.filterwarnings(default_to_pandas_ignore_string)
+pytestmark = [
+    pytest.mark.filterwarnings(default_to_pandas_ignore_string),
+    # TO MAKE SURE ALL FUTUREWARNINGS ARE CONSIDERED
+    pytest.mark.filterwarnings("error::FutureWarning"),
+    # IGNORE FUTUREWARNINGS MARKS TO CLEANUP OUTPUT
+    pytest.mark.filterwarnings(
```

Comment:
could you please elaborate on why this is here?

Reply:
In order not to filter warnings at the test level, there can be many such places (so that an exception does not occur). If you look at the mark above, it causes the `pytest` to throw an exception if a warning of a certain category occurs. I added this mark so that when a new warning occurs from the pandas, we can do the appropriate behavior in modin.